frontend/src/services/getRandomColorCode.js

```diff
@@ -0,0 +1,10 @@
+const getRandomColorCode = () => {
+    let makeColorCode = '0123456789ABCDEF';
```

Comment:
```suggestion
    const makeColorCode = '0123456789ABCDEF';
```
and can be defined outside of the function.

Reply:
changed it. but what's the benefit of defining It outside of the function, if I'm not using it anywhere else?